imageroot/actions/create-module/20postgrestoken

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2025 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import agent
+import os
+import string
+import secrets
+
+try:
+    secrets_env = agent.read_envfile("secrets.env")
+except FileNotFoundError:
+    # Generate a new API token
+    alphabet = string.ascii_letters + string.digits
+    postgres_token = ''.join([secrets.choice(alphabet) for i in range(32)])
+    secrets_env = {
+        "POSTGRES_TOKEN": postgres_token,
```

Comment:
Naming the variable consistently with that used by the container helps to understand the scope.
Also, I would suggest renaming the Python variables accordingly. 
```suggestion
        "POSTGRES_PASSWORD": postgres_token,
```